components/TopHeader.tsx

```diff
@@ -1,26 +1,103 @@
 import React from "react";
 import styles from "../styles/TopHeader.module.css";
+import { Link as ReactScrollLink } from "react-scroll";
+import Image from "next/image";
 
 export default function TopHeader() {
 	return (
 		<div className={styles.globalTopHeader}>
-			<div className={styles.medexLogo}>Medex</div>
-			<div className={styles.topRight}>
-				<span>FAQ</span>
-				<span>spanlans</span>
-				<span className={styles.borderWhite}>Try it for free</span>
-				<span>Log in</span>
+			<div className={styles.topRow}>
```

Comment:
tylko tylko wspomnę, że (module css) to lepsze rozwiązanie niż inlined css, global import css, a nawet czasem może lepsze niż styled-components, niemniej unikałbym tego globalTopHeader

rozwiązania jeszcze lepsze to 
`babel-react-scoped-css` i `styled-jsx`, ale to jest ok